js/client-chat-tournament.js

```diff
@@ -307,7 +307,9 @@
 					if (leaves.length > 0) message.push(arrayToPhrase(leaves) + " left the tournament");
 					this.$lastJoinLeaveMessage.text(message.join("; ") + ".");
 					break;
-
+				case 'replace':
```

Comment:
Looks like the surrounding code double spaces cases.

```suggestion

				case 'replace':
```